modules/doc/content/application_usage/restart_recover.md

```diff
@@ -5,7 +5,7 @@
 - +Restart+: Running a simulation that uses data from a previous simulation. Data in this context is very broad, it can mean spatial field data, non-spatial variables or postprocessors, or stateful object data. Usually the previous and new simulations use different input files.
 - +Recover+: Resuming an existing simulation either due to a fault or other premature termination.
 - +Solution File+: A mesh format containing field data in addition to the mesh (i.e. a normal output file).
-- +Checkpoint+: A snapshot of the simulation data including all meshes, solutions, and stateful object data. Typically one checkpoint is stored in several different files.
+- +Checkpoint+: A snapshot of the simulation data including all meshes, solutions, and stateful object data. Typically one checkpoint is stored in several different files. A checkpoint is always created at every test, but isn't written to by default.
```

Comment:
I'm not sure your comment really helps here. These are general definitions, not really meant to explain functionality. I'd argue that we create a checkpoint "instance" in every "simulation", but we don't always right them out. However, that begs the question, why not? Why create one if you don't use it? This whole discussion should be saved for where you explain the signal system (below).